doc/topics/releases/2015.8.4.rst

```diff
@@ -0,0 +1,15 @@
+===========================
+Salt 2015.8.4 Release Notes
+===========================
+
+Core Changes
+============
+
+- Added ``state.validate`` feature.  This checks SLS data against the salt
+  state API and reports errors if the argument types do not match the types
```

Comment:
Understood. My concern is simply that we're careful about merging it until it can reliably fulfill the promises that it's making. At the very least, my strong preference is to mark it as experimental and log warning messages as such while we flesh out the edge cases.